tests/common/UrlGenerators.h

```diff
@@ -22,50 +22,26 @@
 #include <string>
 #include <vector>
 
-#include "generated/model/Api.h"
-#include "olp/dataservice/read/model/Partitions.h"
-#include "olp/dataservice/read/model/VersionResponse.h"
+#include <olp/dataservice/read/PartitionsRequest.h>
+#include <olp/dataservice/read/model/VersionResponse.h>
 
-namespace mock {
-std::string GenerateGetPartitionsPath(
-    const std::string& layer,
-    const olp::dataservice::read::PartitionsRequest::PartitionIds& partitions,
-    uint64_t version) {
-  std::string path = "/layers/" + layer + "/partitions?";
-  for (const auto& partition : partitions) {
-    path.append("partition=" + partition + "&");
-  }
-  path.append("version=" + std::to_string(version));
-  return path;
-}
+class PathGenerator {
+ public:
```

Comment:
Class and file naming is confusing. Lets name both `UrlGenerator` or `PathGenerator`.